firebase-perf/src/test/java/com/google/firebase/perf/session/PerfSessionTest.java

```diff
@@ -62,7 +62,7 @@ public void setUp() {
 
   @Test
   public void instanceCreation() {
-    PerfSession session = new PerfSession("sessionId", mockClock);
+    PerfSession session = new PerfSession("sessionId", mockClock, true);
```

Comment:
See comment in `FirebasePerfSessionSubscriber`. I'm not sure  passing this in the constructor is a good idea right now - but I will defer it to @mrober .